ttnn/cpp/ttnn/tensor/tensor_impl.hpp

```diff
@@ -51,112 +51,6 @@ std::vector<OutputDataType> cast_vec(const BufferType<InputDataType>& data_to_co
     return converted_data;
 }
 
-// TODO(arakhmati): Should pack_vec_into_uint32_vec be a generator?
-template <typename DataType, template <typename> typename BufferType>
-std::vector<uint32_t> pack_vec_into_uint32_vec(const BufferType<DataType>& data_to_pack) {
```

Comment:
ha, nice.

this was basically replaced with reinterpret_cast?

Reply:
Yes, there were a lot of op tests which went through this flow